homeassistant/components/nuki/lock.py

```diff
@@ -64,13 +64,9 @@ async def async_setup_entry(
 class NukiDeviceEntity(NukiEntity, LockEntity, ABC):
     """Representation of a Nuki device."""
 
+    _attr_has_entity_name = True
```

Comment:
As the lock is the [main function](https://developers.home-assistant.io/docs/core/entity/#example-of-a-switch-entity-which-is-the-main-feature-of-a-device) it's missing `_attr_name = None`

Reply:
None is the default value, without defining it naming is right.